File: app/src/main/java/com/example/myapplication/bean/UserInfo.java
package com.example.myapplication.bean;

public class UserInfo {
    private String record;
    private String name;

    public UserInfo(String record, String name) {
        this.record = record;
        this.name = name;
    }

    public String getRecord() {
        return record;
    }

    public void setRecord(String record) {
        this.record = record;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/com/example/myapplication/room/Entity/UserInfo.kt
package com.example.myapplication.room.Entity

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "user_info")
data class UserInfo (@PrimaryKey @ColumnInfo(name = "userName") val userName: String,
                         @ColumnInfo(name="password")val password:String)